scripts/branch_rename_migration.py

```
@@ -14,19 +14,17 @@
-        print(f"Error: {e.stderr}")
-        return None
+
 
 def get_local_branches():
     """Get list of local branches."""
-    output = run_command("git branch")
+    try:
+        result = subprocess.run(["git", "branch"], shell=False, capture_output=True, text=True, check=True)
+        output = result.stdout.strip()
+    except subprocess.CalledProcessError as e:
+        print(f"Error running command: git branch")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)